backend/app/commands.py

```
@@ -0,0 +1,52 @@
+from app import db
+from app.models.models import VimCommand
+
+def init_commands():
+    """初期コマンドをデータベースに追加"""
+    commands = [
+        # レベル1：基本的な移動
```

Ruff (0.8.2)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)